index.js

```diff
@@ -1,24 +1,35 @@
 /* jshint node: true */
 'use strict';
 
+var path = require('path');
+var Funnel = require('broccoli-funnel');
+var MergeTrees = require('broccoli-merge-trees');
+
 module.exports = {
   name: 'ember-sinon',
 
-  options: {
-    nodeAssets: {
-      'sinon': {
-        import: [{ path: 'pkg/sinon.js', type: 'test' }]
-      }
+  included: function(app) {
+    this._super.included.apply(this, arguments);
+
+    while (typeof app.import !== 'function' && app.app) {
```

Comment:
Should use `this.import` if present instead of this looping.  See https://github.com/rwjblue/ember-runtime-enumerable-includes-polyfill/blob/master/index.js#L12-L16 for an example.